src/components/AddComment.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { postComment } from '../api';
+
+export default function AddComment({ article_id, onAddComment }) {
+    const [commentBody, setCommentBody] = useState('');
+    const [error, setError] = useState(null);
+    const [successMessage, setSuccessMessage] = useState(null);
+
+    const user = 'tickle122'; //Hardcoded user
+    const max_comment_length = 500;
```

Comment:
As you are limiting the comment length some feedback to the user would be a good touch